guides/common/modules/proc_importing-a-subscription-manifest-into-foreman-server.adoc

```diff
@@ -13,6 +13,9 @@ Simple Content Access (SCA) is set on the organization, not the manifest.
 Importing a manifest does not change your organization's Simple Content Access status.
 ====
 
+Simple Content Access simplifies the entitlement experience for administrators.
```

Comment:
```suggestion
Simple Content Access simplifies the subscription experience for administrators.
```
I believe that entitlement-based subscriptions have been removed, so this "entitlement experience" may be an incorrect term. Please, double check with a SME.

Reply:
I've reached out to a SME to confirm.

Reply:
It's confusing because the original [Simple Content Access](https://access.redhat.com/articles/simple-content-access) article on the portal uses the term "entitlement experience" and "experience tooling". The epics use the same terminology.

However, the linked article uses "Subscription Management" in the title. Also, the link appears in [Managing RH subscriptions](https://docs.redhat.com/en/documentation/red_hat_satellite/6.16/html/managing_content/managing_red_hat_subscriptions_content-management). I think it would be less confusing to use "subscription experience".

Reply:
+1 for "subscription experience." Moving away from the word "entitlement" feels right to me.